[CI] HdfsRepositoryAnalysisRestIT class failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #4063 / amazonlinux-2_platform-support-aws](https://gradle-enterprise.elastic.co/s/p6eljof5epmvq)
- [elasticsearch-periodic-platform-support #4019 / amazonlinux-2_platform-support-aws](https://gradle-enterprise.elastic.co/s/az2vw6z42xh4o)
- [elasticsearch-pull-request #30901 / part-2](https://gradle-enterprise.elastic.co/s/elo6amlra5usu)

**Reproduction Line:**
```
undefined
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.repositories.blobstore.testkit.analyze.HdfsRepositoryAnalysisRestIT'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.repositories.blobstore.testkit.analyze.HdfsRepositoryAnalysisRestIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
undefined
```

**Issue Reasons:**
- [main] 3 failures in class org.elasticsearch.repositories.blobstore.testkit.analyze.HdfsRepositoryAnalysisRestIT (0.5% fail rate in 636 executions)
- [main] 2 failures in step amazonlinux-2_platform-support-aws (13.3% fail rate in 15 executions)
- [main] 2 failures in pipeline elasticsearch-periodic-platform-support (12.5% fail rate in 16 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

The other CI failure is on a PR and the error is

> Caused by: java.lang.RuntimeException: Unable to locate plugin 'test-features'. Ensure you've added the following to the build script for project ':x-pack:plugin:snapshot-repo-test-kit:qa:hdfs':

which suggests local invalid configuration. 

I raised https://github.com/elastic/elasticsearch/pull/112906 to unmute the test.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -98,8 +98,6 @@ tests:
 - class: org.elasticsearch.xpack.sql.qa.single_node.JdbcSqlSpecIT
   method: test {case-functions.testUcaseInline3}
   issue: https://github.com/elastic/elasticsearch/issues/112643
-- class: org.elasticsearch.repositories.blobstore.testkit.analyze.HdfsRepositoryAnalysisRestIT
-  issue: https://github.com/elastic/elasticsearch/issues/112889
 - class: org.elasticsearch.xpack.sql.qa.security.JdbcSqlSpecIT
   method: test {case-functions.testUcaseInline1}
   issue: https://github.com/elastic/elasticsearch/issues/112641

